bow-meta.xcodeproj/xcuserdata/tomasruizlopez.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -15,5 +15,18 @@
 			<integer>0</integer>
 		</dict>
 	</dict>
+	<key>SuppressBuildableAutocreation</key>
```

Comment:
You should not upload any file in your `.xcuserdatad` folder, could you remove it from the repository and add a new rule in the `.gitignore` file

Reply:
Added to gitignore and removed.